spec/registry.md

```diff
@@ -374,12 +374,12 @@ All other values produce a _Bad Operand_ error.
 ### Digit Size Options
 
 Some _options_ of number _functions_ are defined to take a "digit size option".
-Implementations of number _functions_ use these _options_ to control aspects of numeric display
+_Function handlers_ of number _functions_ use these _options_ to control aspects of numeric display
```

Comment:
Perhaps "The _function handlers_ for..."?

Reply:
```suggestion
The _function handlers_ for number _functions_ use these _options_ to control aspects of numeric display
```